Assets/HoloToolkit/Input/Scripts/Focus/IFocusTarget.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+using HoloToolkit.Unity.InputModule;
+using System.Collections.Generic;
+
+public interface IFocusTarget : IEventSystemHandler
```

Comment:
Why not keep to the usual naming conventions and make this `IFocusHandler`?

Reply:
Being the _target_ of focus is different than a thing _handling_ focus. It sounds like a pointless differentiation now but once you break apart Focusers and Pointers we found it makes things clearer conceptually.

Also, the behavior of a FocusTarget is different enough from an old FocusHandler that we didn't want to invite confusion by calling them the same thing.